networking_ccloud/tools/netbox_config_gen.py

```diff
@@ -375,6 +376,16 @@ def get_switchgroup_attributes(cls, devices: List[NbRecord]) -> Dict[str, str]:
                 raise ConfigException(f'{device.name} must have a site')
             azs.add(device.site.slug)
 
+            # segment ranges
+            if device.config_context:
+                vlan_range = device.config_context.get('cc', {}) \
+                                                  .get('net', {}) \
+                                                  .get('evpn', {}) \
+                                                  .get('tenant-vlan-range', {})
+                if vlan_range:
+                    vlan_range = tuple(sorted(vlan_range))
+                    vlan_ranges.add(vlan_range)
```

Comment:
I updated the docs to they are `:` separated. I would not do any sanitizing in the config generator.

Reply:
Aye, I'm not against that. But if the config_context is broken (e.g. not a list) this would lead to an exception and we probably don't know where it comes from (no "switch x has broken context"), but that'll only happen if people put in garbage into the context.